modules/compute/src/integrationTest/java/org/apache/ignite/internal/compute/ItWorkerShutdownTest.java

```diff
@@ -370,7 +371,7 @@ private TestingJobExecution<String> executeGlobalInteractiveJob(IgniteImpl entry
     private void createReplicatedTestTableWithOneRow() {
         // Number of replicas == number of nodes and number of partitions == 1. This gives us the majority on primary replica stop.
         // After the primary replica is stopped we still be able to select new primary replica selected.
-        executeSql("CREATE ZONE TEST_ZONE WITH REPLICAS=3, PARTITIONS=1");
+        executeSql("CREATE ZONE TEST_ZONE WITH REPLICAS=3, PARTITIONS=1, STORAGE_PROFILES='" + DEFAULT_STORAGE_PROFILE + "'");
```

Comment:
Do we still need to specify the default profile?

Reply:
Yes, zone's DDL without STORAGE_PROFILES param is not valid at all. We always need to set the zone's profiles manually, there are no any cluster wide default storage profiles.